packages/wallet-adapter-react/src/WalletProvider.tsx

```diff
@@ -62,7 +62,7 @@ export const AptosWalletAdapterProvider: FC<AptosWalletProviderProps> = ({
       setIsLoading(true);
       await walletCore.connect(walletName);
     } catch (error: any) {
-      console.log("connect error", error);
+      console.log(`connect error ${JSON.stringify(error)}`);
```

Comment:
might be b/c we expect the message to be either a string or contains a `message` prop if its an object
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/WalletCore.ts#L240

Reply:
Okay, then we need to make it handle the message prop then or is it supposed to automatically work like this?

Reply:
ideally wallets would have a message prop - which I believe all have except Petra https://www.notion.so/aptoslabs/Wallets-Audit-in-the-Adapter-7-3-2023-cc6bfb1316394be79bb01eeea7ab1d21?showMoveTo=true